### DISCUSSION TITLE
Guide needed!

### DISCUSSION THREAD

**Discussion by dimanranawaka** [intellij-solidity/intellij-solidity#410] [Q&A] [answered]
how to use this plugin? I mean, any step by step guide?

**Comment by SerCeMan** [comment, marked as answer]
Hi, @dimanranawaka!

The goal of the plugin is to primarily _support_ existing Solidity development flows, so it can't be used on its own. It would be helpful if you could describe what you're trying to achieve, your end goal, as it'll make it clearer what type of guide you're looking for.
